static/sass/custom/project.scss

```diff
@@ -28,4 +22,10 @@
       font-style: italic;
       margin: 5px 5px 10px 5px;
     }
+
```

Comment:
Can we make this global? I think we probably could add this require on other forms eventually.

Reply:
Yes, agreed. :) https://github.com/bloomberg/pybossa-default-theme/pull/214/commits/8954cc49372345b47b7066176624317048914086